scripts/data/build_hardcoded.py

```diff
@@ -0,0 +1,354 @@
+import argparse
```

Comment:
Maybe add a high level description of what the file does at the beginning of the file?